vavr/src/main/java/io/vavr/control/Validation.java

```diff
@@ -393,36 +402,36 @@ default boolean isEmpty() {
     T get();
 
     /**
-     * Gets the value if it is a Valid or an value calculated from the error
+     * Gets the value if it is a Valid or an value calculated from the errors
      *
-     * @param other a function which converts an error to an alternative value
+     * @param other a function which converts an errors to an alternative value
      * @return the value, if the underlying Validation is a Valid, or else the alternative value
-     * provided by {@code other} by applying the error.
+     * provided by {@code other} by applying the errors.
      */
-    default T getOrElseGet(Function<? super E, ? extends T> other) {
+    default T getOrElseGet(Function<Seq<? extends E>, ? extends T> other) {
```

Comment:
The 1st generic type param is in contra-variant (= consumer) position, it should have the following generic type bounds:

```java
default T getOrElseGet(Function<? super Seq<? super E>, ? extends T> other) {
```